frontend/assets/defaultcompanyicon.svg

```diff
@@ -0,0 +1,54 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I would move all png / svg eg. files to image folders like `assets/images`

Reply:
Okay, it's done, thanks!